conf/fluent.conf

```diff
@@ -1,26 +1,4 @@
 @include systemd.conf
 @include kubernetes.conf
 
-<match **>
-   @type elasticsearch
-   @id out_es
-   @log_level info
-   include_tag_key true
-   host "#{ENV['FLUENT_ELASTICSEARCH_HOST']}"
-   port "#{ENV['FLUENT_ELASTICSEARCH_PORT']}"
-   scheme "#{ENV['FLUENT_ELASTICSEARCH_SCHEME'] || 'http'}"
-   ssl_verify "#{ENV['FLUENT_ELASTICSEARCH_SSL_VERIFY'] || 'true'}"
-   #user "#{ENV['FLUENT_ELASTICSEARCH_USER']}"
-   #password "#{ENV['FLUENT_ELASTICSEARCH_PASSWORD']}"
-   reload_connections "#{ENV['FLUENT_ELASTICSEARCH_RELOAD_CONNECTIONS'] || 'true'}"
-   logstash_prefix "#{ENV['FLUENT_ELASTICSEARCH_LOGSTASH_PREFIX'] || 'logstash'}"
-   logstash_format true
-   <buffer>
-     flush_thread_count 8
-     flush_interval 5s
-     chunk_limit_size 2M
-     queue_limit_length 32
-     retry_max_interval 30
-     retry_forever true
-   </buffer>
-</match>
\ No newline at end of file
+@include kube-gen.conf
```

Comment:
you forgot to add **elasticsearch-out.conf** ? @waseem-h

Reply:
It is included in kube-gen.conf.tpl

Reply:
Moved it to fluentd.conf